scanapi/tree/request_node.py

```diff
@@ -115,6 +115,10 @@ def retry(self):
         return self.spec.get(RETRY_KEY)
 
     def run(self):
+        """Run request and yours tests.
```

Comment:
@hebertjulio how about "Make HTTP requests and generating test results for the given URLs" ?